### PR TITLE
fluentd updates for ES/Cloudwatch

### PR DESCRIPTION
**Pull request by frankcash**
## Which issue this PR fixes

#131 #133 

## Summary of changes

- Resolves issue with the es updates
- Removes `USER fluentd` as this was causing file mount to be unaccessible - was throwing errors with this kept in
- Reintroduces cloudwatch plugin 

## Images are updated by this PR

- [ ] alertmanager
- [ ] blackbox-exporter
- [ ] configmap-reloader
- [ ] curator
- [ ] elasticsearch
- [ ] elasticsearch-exporter
- [x] fluentd
- [ ] grafana
- [ ] keda
- [ ] keda-metrics-apiserver
- [ ] kibana
- [ ] kube-state
- [ ] kubed
- [ ] nats-server
- [ ] nats-streaming
- [ ] nginx
- [ ] nginx-es
- [ ] node-exporter
- [ ] pgbouncer
- [ ] pgbouncer-exporter
- [ ] postgres-exporter
- [ ] prometheus
- [ ] redis
- [ ] registry
- [ ] statsd-exporter

## Checklist

Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.

- [x] version.txt is updated in the changed image(s)

If a security scan fails for an unchanged image...

- [ ] a fix has been applied OR...
- [ ] an [issue]() has been created

If adding a new image:

- [ ] the directory has the same name you intend it to be published as, less a preceding "ap-"
- [ ] the directory includes a file "version.txt", with version matching the underlying software version
- [ ] the directory includes a file "cve-whitelist.yaml", which may be empty
- [ ] the file .github/PULL_REQUEST_TEMPLATE.md is updated to include the image in the checklist
- [ ] execute the script .circleci/generate_circleci_config.py, commit changes to .circleci/config.yml

## Additional notes for your reviewer
Pods running off of commit https://github.com/astronomer/ap-vendor/commit/a10df172a633b834b518df53df5160ece1e39abb
Screen Shot 2021-08-16 at 2 50 58 PM

Writing to both ES and CW with custom configmap changes to add CW section.  Without CW section configmap runs just fine.

Screen Shot 2021-08-16 at 2 57 52 PM

### CODE REVIEW

File: fluentd/Dockerfile [hunk 1]
```diff
@@ -76,6 +76,4 @@ RUN mkdir -p /var/log/journal
 # to prevent a race condition
 COPY include/wait-for-istio-proxy.rb /wait-for-istio-proxy.rb
 
-USER fluent
```
Comment (danielhoherd) [inline fluentd/Dockerfile:79]: Is it necessary to remove this? This was just added like 12 days ago.

File: fluentd/include/Gemfile [hunk 2]
```diff
@@ -5,10 +5,12 @@ source 'https://rubygems.org'
 
 ### Upstream packages
 gem 'activesupport', '~>6.1'
-gem 'elasticsearch-xpack', '~>7.9'
+gem 'elasticsearch', '7.13.3'
```
Comment (danielhoherd) [inline fluentd/include/Gemfile:8]: Can we pin this more loosely?

Comment (frankcash) [reply fluentd/include/Gemfile:8]: Sure

File: fluentd/Dockerfile (hunk 1 above)
Comment (frankcash) [reply fluentd/Dockerfile:79]: Followed up offline:  it does seem necessary.

Comment (danielhoherd) [reply fluentd/Dockerfile:79]: Link to platform-pvt discussion about this, agreeing to revert. https://astronomer.slack.com/archives/C021EE4HNQJ/p1629146778009900

**Review by danielhoherd — APPROVED** [review]

**Comment by frankcash** [comment]
Pulled `quay.io/astronomer/ap-fluentd:1.13.3-3` and was able to verify it ran properly in my deployment ✅

**Pull request merged**